dockerfile.prod

```diff
@@ -0,0 +1,54 @@
+# Using python2 as the base image
+FROM python:2 as base 
+
+# Create the folder spdx and cd to it
+WORKDIR /spdx
+
+# Copy the requirements.txt file into the container
+COPY requirements.txt .
+
+# Install the python dependencies
+RUN pip install -r requirements.txt
+
+# -----------------------------
+
+# Container to lint python code
+FROM base as linter 
+
+# Setting current work directory to /spdx
+WORKDIR /spdx
+
+# Copy all files into the container
+COPY . .
+
+# Installing pylint and linting the python code
+RUN pip install pylint && pylint -E *.py **/*.py || printf "\n SPDX online tools has got some errors! Continuing the build for now\n\n"
```

Comment:
What if we add pylint or any other linter in the GitHub actions and make sure each pull request that we merge is error-free? That way I don't think we would have to check it here.

What do you think?